<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaptisForm</class>
 <widget class="QWidget" name="BaptisForm">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>560</width><height>420</height></rect>
  </property>
  <property name="windowTitle">
   <string>BaptisForm (Sederhana)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id_baptis">
       <property name="text"><string>Id Baptis</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="id_baptis">
       <property name="enabled"><bool>false</bool></property>
       <property name="maximum"><number>2147483647</number></property>
      </widget>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="label_id_jemaat">
       <property name="text"><string>Id Jemaat</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="id_jemaat"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="label_nama_ayah">
       <property name="text"><string>Nama Ayah</string></property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nama_ayah"/>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="label_nama_ibu">
       <property name="text"><string>Nama Ibu</string></property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nama_ibu"/>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="label_nama_wali">
       <property name="text"><string>Nama Wali</string></property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="nama_wali"/>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="label_tgl_baptis">
       <property name="text"><string>Tgl Baptis</string></property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="tgl_baptis">
       <property name="calendarPopup"><bool>true</bool></property>
      </widget>
     </item>

     <item row="6" column="0">
      <widget class="QLabel" name="label_pelayan">
       <property name="text"><string>Pelayan</string></property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="pelayan"/>
     </item>

     <item row="7" column="0">
      <widget class="QLabel" name="label_surat_baptis">
       <property name="text"><string>Surat Baptis</string></property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="surat_baptis"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Simpan</string></property></widget></item>
     <item><widget class="QPushButton" name="btnClear"><property name="text"><string>Bersihkan</string></property></widget></item>
     <item>
      <spacer name="hspacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
     <item><widget class="QPushButton" name="btnClose"><property name="text"><string>Tutup</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
